src/PdfSharper/Drawing.Layout/XTextFormatter.cs

```diff
@@ -118,50 +150,66 @@ public XParagraphAlignment Alignment
         /// <param name="layoutRectangle">The layout rectangle.</param>
         public void DrawString(string text, XFont font, XBrush brush, XRect layoutRectangle)
         {
-            DrawString(text, font, brush, layoutRectangle, XStringFormats.TopLeft);
+            this.DrawString(text, false, font, brush, layoutRectangle, XStringFormats.TopLeft);
         }
 
         /// <summary>
         /// Draws the text.
         /// </summary>
-        /// <param name="text">The text to be drawn.</param>
```

Comment:
Keep original comments where appropriate. Name of the game here is the least number of changes.